browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -6225,6 +6493,125 @@ index e0e8809d8f39614b2b6bb8c7cf0f02652d574fe7..5d05504ec57869c9d6c6562c42376400
  {
      auto* display = gdk_display_get_default();
      if (!display)
+diff --git a/Source/WebCore/platform/libwpe/PasteboardLibWPE.cpp b/Source/WebCore/platform/libwpe/PasteboardLibWPE.cpp
+index 78b48a3a892e46b8c150ab277050b0258ef69068..0aeb9a4d203988d7ee08f41ec0d16df3f585aa87 100644
+--- a/Source/WebCore/platform/libwpe/PasteboardLibWPE.cpp
++++ b/Source/WebCore/platform/libwpe/PasteboardLibWPE.cpp
+@@ -32,7 +32,9 @@
+ #include "PasteboardStrategy.h"
+ #include "PlatformStrategies.h"
+ #include <wtf/Optional.h>
+-
++#if ENABLE(DRAG_SUPPORT)
++#include "DragData.h"
++#endif
+ namespace WebCore {
+ 
+ std::unique_ptr<Pasteboard> Pasteboard::createForCopyAndPaste()
+@@ -73,6 +75,9 @@ String Pasteboard::readOrigin()
+ 
+ String Pasteboard::readString(const String& type)
+ {
++    if (m_selectionData)
++        return m_selectionData->text();
```

Comment:
Do we care about other selection data types?

Reply:
yes, fixed!